index.html

```diff
@@ -1 +1,34 @@
+        id="image"
+      />
+      <br />
+      <fig-caption id="img-caption">Image of Albert Einstein</fig-caption>
+    </figure>
+
+    <p id="tribute-info">
+      Albert Einstein was a famous physicist. His research spanned from quantum
+      mechanics to theories about gravity and motion.
+    </p>
+
+    <a
+      href="https://en.wikipedia.org/wiki/Albert_Einstein"
+      target="_blank"
+      id="tribute-link"
+    >
+      Read more
+    </a>
```

Comment:
Your code works great! 👍 
However, you can put these codes on one line

`<a href="https://en.wikipedia.org/wiki/Albert_Einstein" target="_blank" id="tribute-link"> Read more </a>`

It makes your code cleaner and smaller 😃